sys/net/gnrc/doc.txt

```diff
@@ -0,0 +1,172 @@
+/**
+ * @defgroup    net_gnrc Generic (gnrc) network stack.
```

Comment:
Doesn't it make much more sense to put this whole thing in `sys/include/net/gnrc.h`